org/readme.md

```diff
@@ -28,6 +29,44 @@ Organization admins are:
 Github Teams for the TOC, all Working Group Leads, Working Groups and Working Group Areas are generated according to [rfc-0014-github-teams-and-access.md](https://github.com/cloudfoundry/community/blob/main/toc/rfc/rfc-0014-github-teams-and-access.md).
 Repositories listed in the working group yaml block that belong to github organizations other than `cloudfoundry` are ignored.
 
+### Branch Protection Rules
+
+Working groups can opt-in into branch protection rule generation for their projects according to [rfc-0015-branch-protection](https://github.com/cloudfoundry/community/blob/main/toc/rfc/rfc-0015-branch-protection.md) by setting a configuration flag in the working group charter yaml:
+
+```
+config:
+  generate_rfc0015_branch_protection_rules: true
+```
+
+Branch protection rules are applied using the [branchprotector tool from the prow toolset](https://docs.prow.k8s.io/docs/components/optional/branchprotector/).
+Rules specified in [branchprotection.yml](https://github.com/cloudfoundry/community/blob/main/org/branchprotection.yml) take precedence, i.e. no RFC-0015 rules are generated for repositories listed here but the static configuration is taken without modification.
+
+The generated branch protection rules specification for working group projects look like:
+```
+branch-protection:
+  orgs:
+    cloudfoundry:
+      repos:
+        # automation generates config for all repos belonging to a WG unless an explicit configuration exists in branchprotection.yml
+        <projectname>:
+          protect: true
+          enforce_admins: true
+          allow_force_pushes: false
+          allow_deletions: false
+          allow_disabled_policies: true  # needed to allow branches w/o branch protection
+          required_pull_request_reviews:
+            dismiss_stale_reviews: true
+            require_code_owner_reviews: true
```

Comment:
What happens if no code owners have been configured for a repo?

Reply:
https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/managing-a-branch-protection-rule

> Optionally, to require review from a code owner when the pull request affects code that has a designated owner, select **Require review from Code Owners**.

This flag has only affect, if code owners have been configured.

Tested with https://github.com/cloudfoundry/branchprotection-test/pull/1